### PR TITLE
docs: Add  basic plugins doc

### PR DESCRIPTION
**Pull request by Gabriel-Trintinalia**
This PR adds basic plugins doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,9 +159,9 @@ ______________________________________________________________________
...
 ```
 
-## Continuous Tracing
+## Plugins
 
-The continuous tracing plugin is documented [here](CONTINUOUS_TRACING.md).
+Plugins are documented [here](PLUGINS.md).
 
 ## Release Process
 Here are the steps for releasing a new version of the plugin:
```
Comment (pinges) [inline README.md:167]:
```suggestion
Here are the steps for releasing a new version of the plugins:
```

File: PLUGINS.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+environment variable. An environment variable was chosen instead of a command line flag to avoid leaking the webhook URL
+in the process list.
+
+The environment variable can be set via systemd using the following command:
+
+```shell
+Environment=SLACK_SHADOW_NODE_WEBHOOK_URL=https://hooks.slack.com/services/SECRET_VALUES
+```
+
+### Invalid trace handling
```
Comment (pinges) [inline PLUGINS.md:83]: You define what happens for invalid traces, but nothing about what happens if there are no problems.

**Review by pinges — APPROVED** [review]

File: PLUGINS.md (hunk 2 above)
Comment (Gabriel-Trintinalia) [reply PLUGINS.md:83]: that is in the first line of that paragraph, in case of success the trace file is just deleted

File: PLUGINS.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+| DENY_LIST_PATH | lineaDenyList.txt | `--plugin-linea-deny-list-path` |
+| MAX_TX_GAS_LIMIT_OPTION | 30_000_000 | `--plugin-linea-max-tx-gas-limit` |
+| MAX_TX_CALLDATA_SIZE | 60_000 | `--plugin-linea-max-tx-calldata-size` |
+
+## RPC
+
+### Counters - RollupCounterRpcPlugin
+#### `rollup_getTracesCountersByBlockNumberV0` 
+
+The RollupRpcEndpointServicePlugin registers an RPC endpoint named `getTracesCountersByBlockNumberV0`
```
Comment (macfarla) [inline PLUGINS.md:39]:
```suggestion
The RollupCounterRpcPlugin registers an RPC endpoint named `getTracesCountersByBlockNumberV0` 
```

**Pull request merged**